lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -226,7 +226,7 @@ def swap_positions_with(item)
         end
 
         def acts_as_list_list
-          acts_as_list_class.unscope(:select, :where).where(scope_condition)
+          acts_as_list_class.unscoped.where(scope_condition)
```

Comment:
Use `default_scoped` in this case. https://github.com/rails/rails/pull/38319

```suggestion
          acts_as_list_class.default_scoped.unscope(:select, :where).where(scope_condition)
```